diff -uNr Template-Toolkit-2.09-org/lib/Template/Plugin/Image.pm Template-Toolkit-2.09/lib/Template/Plugin/Image.pm
--- Template-Toolkit-2.09-org/lib/Template/Plugin/Image.pm	Wed Apr 23 14:58:29 2003
+++ Template-Toolkit-2.09/lib/Template/Plugin/Image.pm	Mon Jun 30 16:28:12 2003
@@ -43,7 +43,12 @@
             my @stuff = Image::Size::imgsize($file);
             return { "width"  => $stuff[0],
                      "height" => $stuff[1],
-                     "error"  => $stuff[2]
+                     "error"  =>
+                        # imgsize returns either a three letter file type
+                        # or an error message as third value
+                        (defined($stuff[2]) && length($stuff[2]) > 3
+                            ? $stuff[2]
+                            : undef),
                    };
         }
     }
