--- Template-Toolkit-2.20/Makefile.PL~	2008-12-11 14:05:20.000000000 +0100
+++ Template-Toolkit-2.20/Makefile.PL	2008-12-11 14:11:35.126277920 +0100
@@ -160,11 +160,11 @@
 version_check();
 mandatory_modules();
 optional_stash_xs();
-#optional_extras();
-#splash_images();
-#html_docs();
-#html_docstyle();
-#html_examples();
+optional_extras();
+splash_images();
+html_docs();
+html_docstyle();
+html_examples();
 write_defaults();
 
 print "\n";
